src/cpio/client_providers/kms_client_provider/azure/azure_kms_client_provider.h

```diff
@@ -35,8 +37,8 @@ namespace google::scp::cpio::client_providers {
 class AzureKmsClientProvider : public KmsClientProviderInterface {
  public:
   explicit AzureKmsClientProvider(
-      core::HttpClientInterface* http_client,
-      AuthTokenProviderInterface* auth_token_provider)
+      const std::shared_ptr<core::HttpClientInterface>& http_client,
```

Comment:
This does not appear to be consistent with the new interface definition in 3.4.0.

Reply:
fixed